adapters/primitive-adapters/src/main/kotlin/com/squareup/sqldelight/adapter/primitive/BooleanColumnAdapter.kt

```diff
@@ -0,0 +1,9 @@
+package com.squareup.sqldelight.adapter.primitive
```

Comment:
So this probably should be `com.squareup.sqldelight.primitive.adapter` if we're following the approach of the other modules (e.g., `JdbcDriver` has the import `com.squareup.sqldelight.jdbc.driver`), but personally I'd find the grouping in the imports to make a lot more sense here than having the adapter name first (`com.squareup.sqldelight.adapter.primitive`).

Reply:
similar to above, I'd say keep whatever you've already done and then we can change when we split out the dialects